mapviz_plugins/src/odometry_plugin.cpp

```diff
@@ -377,6 +422,17 @@ namespace mapviz_plugins
       ui_.arrow_size->setValue(node["arrow_size"].as<int>());
     }
 
+    if (node["show_timestamps"])
```

Comment:
It occurs to me that this PR may depend on #493, in which case that should be explicitly stated in the PR text so we can review the PRs in order.